test/parsing_test.js

```diff
@@ -245,12 +241,12 @@ vows
         return Cookie.parse("\x08", true) || null;
       },
       "doesn't parse": function(c) {
-        assert.equal(c, null);
+        assert.equal(c, undefined);
```

Comment:
You missed the `Cookie.parse(...) || null` above which means the value of `c` here is actually `null` but the test still passes since `assert.equal` is used instead of `assert.strictEqual`.

```suggestion
        assert.equal(c, SHOULD_BE_UNDEFINED);
```